src/components/UpgradeTableOfContents.tsx

```diff
@@ -145,11 +115,28 @@ const UpgradeTableOfContents: React.FC<IProps> = ({
   }
 
   return (
-    <Aside className={className}>
-      <OuterList>
+    <Box
+      as="aside"
+      className={className}
```

Comment:
I don't think we are passing any `className` to this component in any of the uses. I would recommend removing it since we should handle all the styles using props.